presto-parquet/src/main/java/com/facebook/presto/parquet/reader/PageReader.java

```diff
@@ -112,4 +134,9 @@ public DictionaryPage readDictionaryPage()
             throw new RuntimeException("Error reading dictionary page", e);
         }
     }
+
+    public static long getFirstRowIndex(int pageIndex, OffsetIndex offsetIndex)
```

Comment:
this function only used once in this class, let's inline it

Reply:
It is also used in ParquetColumnChunk.java

Reply:
oh, OK